src/multi-selector.js

```diff
@@ -79,32 +80,49 @@ const MultiSelector = React.createClass({
 			activeIndex: index,
 			mouseIndex: null,
 		}, () => {
-			this.searchItems[this.state.activeIndex].scrollIntoView();
-			if (this.state.mouseActive) {
-				this.setState({
-					mouseActive: false,
-					mouseFunc: () => {
-						this.setState({
-							mouseActive: true
-						})
-						document.removeEventListener("mousemove", this.state.mouseFunc);
-					}
-				}, () => {
-					document.addEventListener("mousemove", this.state.mouseFunc);
-				})
+			if (!isNull(this.state.activeIndex) && !isNull(this.searchItems[this.state.activeIndex])) {
```

Comment:
Would this.searchItems[this.state.activeIndex] be null? I might be wrong but it seems like it would be undefined